plugins/module_utils/foreman_helper.py

```diff
@@ -617,6 +617,15 @@ def fail_from_exception(self, exc, msg):
                 fail['error'] = exc.response.text
         self.fail_json(**fail)
 
+    def handle_organization_param(self, entity_dict):
```

Comment:
this is in `ForemanAnsibleModule` because that's what's shared between all the Katello modules and foreman_hostgroup. We could move that to `KatelloAnsibleModule` and not add it in hostgroup.

Reply:
I think doing katello only is cleaner, but wanted to hear your opinion.

Reply:
So an OrganizationMixin would also work. But i am fine with katello only.